File: 2239.cpp
class Solution {
public:
    int findClosestNumber(vector<int>& nums) {
        int res = nums[0];

        for(int i = 0; i < nums.size(); i++) {
            if(abs(res) > abs(nums[i])) {
                res = nums[i];
            } else if(abs(res) == abs(nums[i])) {
                res = fmax(res, nums[i]);
            }
        }
        return res;
    }
};

File: 2239.py
class Solution:
    def findClosestNumber(self, nums: List[int]) -> int:
        res = nums[0]

        for num in nums:
            if(abs(res) > abs(num)):
                res = num
            elif(abs(res) == abs(num)):
                res = max(res, num)
        return res